README.md

```diff
@@ -11,7 +11,9 @@ This project is a simple Rails app intended to help you demonstrate:
 * Problem-solving abilities
 * Ability to use the [Git version control system](https://git-scm.com/)
 
-Your goal is to add functionality and fix bugs so that the app's [RSpec test suite](spec/) passes.
+## Scenario
+
+Your goal is to add functionality and fix bugs so that the app's [RSpec test suite](spec/) passes. This will involve modifying both application code (where functionality is missing or broken) and RSpec code (where the functionality is correct, but the test is not written correctly).
```

Comment:
Didn't we change this to be that all the specs are correct, so they won't have to guess whether it is a spec issue or a code issue?

Reply:
I remember discussing that but didn't remember what we decided. I'll make that fix now